readthedocs/donate/views.py

```diff
@@ -54,3 +56,15 @@ def get_queryset(self):
 
     def get_template_names(self):
         return [self.template_name]
+
+
+def click_proxy(request, promo_id):
```

Comment:
Ah, true. Should be able to do: https://docs.djangoproject.com/en/1.9/ref/csrf/#django.views.decorators.csrf.requires_csrf_token perhaps